plugfest-3-2023/signed_badge_example-p3.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://purl.imsglobal.org/spec/ob/v3p0/context.json"
+  ],
+  "id": "urn:uuid:7b2605ae-e89d-4689-9d9b-fe5395e286df",
+  "type": [
+    "VerifiableCredential",
+    "OpenBadgeCredential"
+  ],
+  "name": "JFF x vc-edu PlugFest 3 Interoperability",
+  "issuer": {
+    "type": ["Profile"],
+    "id": "did:key:z6MknTHVAhsRyNzBG6h5wjTrg76UGUaBoXS5Vs9Apfr1fbCj",
+    "name": "Jobs for the Future (JFF) Labs"
+  },
+  "issuanceDate": "2023-10-09T00:00:00Z",
+  "credentialSubject": {
+    "type": ["AchievementSubject"],
+    "id": "did:key:z6MkjnTsAhSmGMxbGrHqGqoubDpdWyPbPTNWpcYBNfWXMXgD",
+    "achievement": {
+      "id": "urn:uuid:ac254bd5-8fad-4bb1-9d29-efd938536926",
+      "type": ["Achievement"],
+      "name": "JFF x vc-edu PlugFest 3 Interoperability",
+      "description": "This wallet supports the use of W3C Verifiable Credentials and has demonstrated interoperability during the presentation request workflow during JFF x VC-EDU PlugFest 3.",
+      "criteria": {
+        "type": "Criteria",
+        "narrative": "Wallet solutions providers earned this badge by demonstrating interoperability during the presentation request workflow. This includes successfully receiving a presentation request, allowing the holder to select at least two types of verifiable credentials to create a verifiable presentation, returning the presentation to the requestor, and passing verification of the presentation and the included credentials."
```

Comment:
```suggestion
        "narrative": "Wallet solution providers earned this badge by demonstrating interoperability during the presentation request workflow. This included successfully receiving a presentation request, allowing the holder to select at least two types of verifiable credentials to create a verifiable presentation, returning the presentation to the requester, and passing verification of the presentation and the included credentials."
```

Reply:
Applied in bb2c6f04a513c31454b844a1bfeaab6769162c85